tests/System.Buffers.ReaderWriter.Tests/ReadableBufferFacts.cs

```diff
@@ -1,196 +0,0 @@
-    {
-        public class Array : ReadableBufferFacts
-        {
-            public Array() : base(ReadOnlyBufferFactory.Array) { }
-        }
-
-        public class OwnedMemory : ReadableBufferFacts
-        {
-            public OwnedMemory() : base(ReadOnlyBufferFactory.OwnedMemory) { }
-        }
-
-        public class SingleSegment : ReadableBufferFacts
-        {
-            public SingleSegment() : base(ReadOnlyBufferFactory.SingleSegment) { }
-        }
-
-        public class SegmentPerByte : ReadableBufferFacts
-        {
-            public SegmentPerByte() : base(ReadOnlyBufferFactory.SegmentPerByte) { }
-        }
-
-        internal ReadOnlyBufferFactory Factory { get; }
-
-        internal ReadableBufferFacts(ReadOnlyBufferFactory factory)
-        {
-            Factory = factory;
-        }
-
-        [Fact]
-        public void EmptyIsCorrect()
-        {
-            var buffer = Factory.CreateOfSize(0);
-            Assert.Equal(0, buffer.Length);
-            Assert.True(buffer.IsEmpty);
-        }
-
-        [Theory]
-        [InlineData(1)]
-        [InlineData(8)]
-        public void LengthIsCorrect(int length)
-        {
-            var buffer = Factory.CreateOfSize(length);
-            Assert.Equal(length, buffer.Length);
-        }
-
-        [Theory]
-        [InlineData(1)]
-        [InlineData(8)]
-        public void ToArrayIsCorrect(int length)
-        {
-            var data = Enumerable.Range(0, length).Select(i => (byte)i).ToArray();
-            var buffer = Factory.CreateWithContent(data);
-            Assert.Equal(length, buffer.Length);
-            Assert.Equal(data, buffer.ToArray());
-        }
-
-        [Theory(Skip = "Need to fix the exception being thrown for various scenarios. Some ReadableBufferFacts throw InvalidOperationException")]
-        [MemberData(nameof(OutOfRangeSliceCases))]
-        public void ReadableBufferDoesNotAllowSlicingOutOfRange(Action<ReadOnlySequence<byte>> fail)
-        {
-            var buffer = Factory.CreateOfSize(100);
-            var ex = Assert.Throws<ArgumentOutOfRangeException>(() => fail(buffer));
-        }
-
-        [Fact]
-        public void ReadableBufferGetPosition_MovesReadCursor()
-        {
-            var buffer = Factory.CreateOfSize(100);
-            var cursor = buffer.GetPosition(65);
-            Assert.Equal(buffer.Slice(65).Start, cursor);
-        }
-
-        [Fact]
-        public void ReadableBufferGetPosition_ChecksBounds()
-        {
-            var buffer = Factory.CreateOfSize(100);
-            Assert.Throws<ArgumentOutOfRangeException>(() => buffer.GetPosition(101));
-        }
-
-        [Fact]
-        public void ReadableBufferGetPosition_DoesNotAlowNegative()
-        {
-            var buffer = Factory.CreateOfSize(20);
-            Assert.Throws<ArgumentOutOfRangeException>(() => buffer.GetPosition(-1));
-        }
-
-        [Fact]
-        public void SegmentStartIsConsideredInBoundsCheck()
```

Comment:
Are we losing some test coverage here (including other tests in this file)?

Reply:
I assume we've moved these tests to CoreFX as they were all tests for ReadOnlySegment

Reply:
Er, ReadOnlySequence

Reply:
Yes, ROS tests can be removed.